docs/api/client_libraries.md

```diff
@@ -6,83 +6,77 @@ nav_order: 1
 
 # Epidata API Client Libraries
 
+To access Delphi Epidata programmatically, we recommend our client libraries:
+
+- R: [epidatr](https://cmu-delphi.github.io/epidatr/),
+- Python: [delphi-epidata](https://pypi.org/project/delphi-epidata/) (soon to be replaced [epidatpy](https://github.com/cmu-delphi/epidatpy)),
```

Comment:
We don't have doc sites for either epidatpy or the javascript library?

```suggestion
- Python: [delphi-epidata](https://pypi.org/project/delphi-epidata/) (soon to be replaced with [epidatpy](https://github.com/cmu-delphi/epidatpy)),
```

Reply:
Doesn't seem like the JS lib has a doc site and not sure how/whether to make that happen. epidatpy has a tiny amount of documentation and it needs to get hosted on https://about.readthedocs.com/ (I have a TODO about it somewhere).